src/cljs/rems/actions/assign_external_id.cljs

```diff
@@ -0,0 +1,59 @@
+(ns rems.actions.assign-external-id
+  (:require [clojure.string :as str]
+            [re-frame.core :as rf]
+            [rems.actions.action :refer [action-button action-form-view action-comment button-wrapper collapse-action-form]]
+            [rems.flash-message :as flash-message]
+            [rems.text :refer [text]]
+            [rems.util :refer [post!]]))
+
+(rf/reg-event-fx
+ ::open-form
+ (fn [{:keys [db]} _]
+   {:db (assoc db ::comment "")}))
```

Comment:
This command doesn't use a comment.

Reply:
Should be `::external-id` because now the form is not reset after submit.

Reply:
done